config/crd/experimental/gateway.networking.k8s.io_grpcroutes.yaml

```diff
@@ -2204,6 +2205,8 @@ spec:
             required:
             - parents
             type: object
+        required:
+        - spec
```

Comment:
@robscott @shaneutt A heads up on this one: I've found that the spec field for grpcroutes wasn't required. Someone could create this object without a spec. I have added as usually specs should always be required, but let me know if my assumption here is wrong

Reply:
I have just confirmed I can create a grpc route without spec before this change:

```
kubectl get grpcroutes.gateway.networking.k8s.io foo-route -o yaml
apiVersion: gateway.networking.k8s.io/v1
kind: GRPCRoute
metadata:
  creationTimestamp: "2025-07-17T22:24:36Z"
  generation: 1
  name: foo-route
  namespace: default
  resourceVersion: "623873"
  uid: 61d7c957-092b-4094-ae3d-a8e8e0cf3a8e
status:
  parents: []
```